basics.rb

```diff
@@ -8,9 +8,11 @@
 post '/payload' do
   push = JSON.parse(request.body.read)
   puts "I got some JSON: #{push.inspect}"
-  jsontest = "I got some JSON: #{push.inspect}"
-  Collaborator.add repo_name: "githubschool/open-enrollment-classes-introduction-to-github", issue_num: 927
-  Collaborator.add repo_name: "githubschool/on-demand-github-pages", issue_num: 1
+
+  Collaborator.add repo_name: push["repository"]["full_name"].to_s, issue_num: push["issue"]["number"].to_i, user_login: push["issue"]["user"]["login"].to_s
+
+  # Collaborator.add repo_name: "githubschool/on-demand-github-pages", issue_num: 1
```

Comment:
I don't know what to do with this, do we leave the `githubschool/on-demand-github-pages` as is?